src/Popper.test.js

```diff
@@ -35,33 +37,41 @@ describe('Popper component', () => {
   it('renders the expected markup', async () => {
     const referenceElement = document.createElement('div');
 
-    const { asFragment } = await renderPopper({ referenceElement });
-
-    await waitFor(() => {
-      expect(asFragment()).toMatchSnapshot();
-    });
+    const { findByTestId } = renderPopper({ referenceElement });
+
+    expect(await findByTestId(TEST_ID)).toMatchInlineSnapshot(`
```

Comment:
Isn't `expect(async thing).resolves.toMatchInlineSnapshot(...)` the "Jest" way?

Reply:
I’ve not come across chained/fluent calls like that in `@testing-library/...`.